doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
+        String _id PK
+        String name
+        String googleID
+        String address
+        ObjectId[] dogs
+    }
+    DOG {
```

Comment:
why only dog? what do you think about calling it PET? if you add a "type" field you can use this entity to store dogs, cats, birds...

Reply:
i can change it to PET, though the specification for this app is exclusive por dogs